dc/core/MovieUpdater.h

```diff
@@ -82,16 +84,37 @@ class MovieUpdater : public DataSource
     /** Increment and synchronize movie timestamp across all wall processes. */
     bool synchronizeTimestamp( WallToWallChannel& channel );
 
+    /**
+     * Allow the updater to prepare the next frame, i.e. a previous
+     * decode & swap of a movie frame is finished.
+     */
+    void requestNextFrame();
+
+    /**
+     * @return true if after synchronizeTimestamp() the tile needs update aka
+     * decode the next movie frame.
+     */
+    bool updateTile() const;
```

Comment:
should be isNextFrameReady() or similar. This is a getter, not an update action, plus the MovieUpdater doesn't know about the concept of Tiles. Also rename the _updateTile member.